includes/Order/Manager.php

```diff
@@ -16,9 +17,9 @@ class Manager {
      *
      * @since 3.0.0
      *
-     * @return void
+     * @return WC_Order[]
      */
-    public function all( $args = [] ) {
+    public function all( $args = [] ): array {
```

Comment:
we are not supporting return type for functions yet. kindly remove the functions return type.